app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Hey @Rounak122 please remove this google-services.json file, as this should not be uploaded to github repo 😬

Reply:
@jayskhatri I have deleted old illustrations and renamed the new ones the same. I am removing google-services.json, I didn't knew that, thanks.

Reply:
@jayskhatri I have done the changes. please check once.